composer.json

```
@@ -6,7 +6,7 @@
         "php": ">=7.4",
         "ext-json": "*",
         "keboola/storage-api-client": "^13.0|^14.0",
-        "symfony/validator": "^5.0|^6.0"
+        "symfony/validator": "^4.0|^5.0|^6.0"
```

Koukam ze v Conneciton mate `^4.4`, hodil bych to i sem, at to neotvirame vic, nez je nutny.

```suggestion
        "symfony/validator": "^4.4|^5.0|^6.0"
```